Python/ceval.c

```diff
@@ -1061,34 +1064,134 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
     Py_DECREF(current_executor);
     goto resume_with_error;
 
+// Jump here from _EXIT_TRACE
+exit_trace:
+    _PyFrame_SetStackPointer(frame, stack_pointer);
+    frame->instr_ptr = next_uop[-1].target + _PyCode_CODE(_PyFrame_GetCode(frame));
```

Comment:
I wish. We're getting here from `_EXIT_TRACE`, and we must use the `target` from that uop. But `_EXIT_TRACE` just jumps here, it doesn't do anything else (and I expect Brandt likes it that way). There is no `_SET_IP` uop preceding `_EXIT_TRACE` any more.

Reply:
`_EXIT_TRACE` is a just a degenerate form of a side exit where the condition is always true. Rather than all the special cases, we should use the `target` field in `_EXIT_TRACE`.